docs/trulens/tracking/logging/where_to_log/index.md

```diff
@@ -14,7 +14,9 @@ For example, for Postgres database `trulens` running on `localhost` with usernam
 
     ```python
     from trulens.core import TruSession
-    session = TruSession(database_url="postgresql://trulensuser:password@localhost/trulens")
+    from trulens.core.database.connector.default import DefaultDBConnector
+    connector = DefaultDBConnector(database_url = "postgresql://trulensuser:password@localhost/trulens")
+    session = TruSession(connector = connector)
     ```
 
 After which you should receive the following message:
```

Comment:
can you update the below message? Should now be TruSession instead of Tru

Reply:
updated